--- a/configure.ac	2015-06-07 14:59:26.958750030 +0000
+++ b/configure.ac	2015-06-07 14:58:55.136751368 +0000
@@ -260,14 +260,11 @@
 dnl - color
 dnl ---------------------------------------------------------------------------
 
-PKG_CHECK_MODULES(COLOR, [colord >= 0.1.27 cinnamon-desktop >= $CINNAMON_DESKTOP_REQUIRED_VERSION libcanberra-gtk3])
 
 dnl ---------------------------------------------------------------------------
 dnl - wacom
 dnl ---------------------------------------------------------------------------
 build_wacom=false
-PKG_CHECK_MODULES(WACOM, [libwacom >= $LIBWACOM_REQUIRED_VERSION x11 xi xtst gudev-1.0 cinnamon-desktop xorg-wacom librsvg-2.0 >= $LIBRSVG_REQUIRED_VERSION gtk+-3.0 >= 3.8.0],
-                  [build_wacom="true" AC_DEFINE(HAVE_WACOM, 1, [Define if wacom is being build])], [build_wacom="false"])
 
 AM_CONDITIONAL(BUILD_WACOM, test "x$build_wacom" = "xtrue")
 
--- a/plugins/Makefile.am	2015-06-07 14:59:26.958750030 +0000
+++ b/plugins/Makefile.am	2015-06-07 14:57:11.625755721 +0000
@@ -6,7 +6,6 @@
 	automount	\
 	background  \
 	clipboard	\
-	color		\
 	cursor		\
 	dummy		\
     datetime    \
