--- a/variety/Util.py	2015-07-17 17:56:20.985115610 +0000
+++ b/variety/Util.py	2015-07-17 18:07:13.470071577 +0000
@@ -25,12 +25,12 @@
 import string
 import threading
 import time
-import pyexiv2
 import urllib
 import urllib2
 from urlparse import urlparse
 from DominantColors import DominantColors
 from gi.repository import Gdk, Pango, GdkPixbuf, GLib
+from gi.repository import GExiv2
 import inspect
 import subprocess
 import platform
@@ -235,13 +235,8 @@
     @staticmethod
     def write_metadata(filename, info):
         try:
-            pyexiv2.xmp.register_namespace("https://launchpad.net/variety/", "variety")
-        except KeyError:
-            pass
-
-        try:
-            m = pyexiv2.ImageMetadata(filename)
-            m.read()
+            m = GExiv2.Metadata(filename)
+            m.register_xmp_namespace("https://launchpad.net/variety/", "variety"
             m["Xmp.variety.info"] = VARIETY_INFO
             for k, v in info.items():
                 if k == 'author':
@@ -261,9 +256,12 @@
                     m["Xmp.variety." + k] = str(v)
                 else:
                     m["Xmp.variety." + k] = v
-            m.write()
+            m.save_file()
             return True
         except Exception:
+            print 'something went wrong writing exif'
+            print e
+  
             # could not write metadata inside file, use json txt instead
             try:
                 with io.open(filename + '.metadata.json', 'w', encoding='utf8') as f:
@@ -276,14 +274,8 @@
     @staticmethod
     def read_metadata(filename):
         try:
-            pyexiv2.xmp.register_namespace("https://launchpad.net/variety/", "variety")
-        except KeyError:
-            pass
-
-        try:
-            m = pyexiv2.ImageMetadata(filename)
-            m.read()
-
+            m = GExiv2.Metadata(filename)
+            m.register_xmp_namespace("https://launchpad.net/variety/", "variety")
             info = {}
             keys = ["sourceName", "sourceLocation", "sourceURL", "sourceType", "imageURL", "author", "authorURL"]
             for k in keys:
@@ -324,6 +316,7 @@
             return info
 
         except Exception, e:
+            print e;
             # could not read metadata inside file, try reading json metadata instead
             try:
                 with io.open(filename + '.metadata.json', encoding='utf8') as f:
@@ -360,9 +353,8 @@
         if rating is not None and (rating < -1 or rating > 5):
             raise ValueError("Rating should be between -1 and 5, or None")
 
-        m = pyexiv2.ImageMetadata(filename)
-        m.read()
-
+        m = Gexiv2.Metadata(filename)
+       
         if rating is None:
             for key in ["Xmp.xmp.Rating", "Exif.Image.Rating", "Exif.Image.RatingPercent"]:
                 if key in m:
@@ -375,12 +367,11 @@
             elif "Exif.Image.RatingPercent" in m:
                 del m["Exif.Image.RatingPercent"]
 
-        m.write()
+        m.save_file()
 
     @staticmethod
     def get_rating(filename):
-        m = pyexiv2.ImageMetadata(filename)
-        m.read()
+        m = GExiv2.Metadata(filename)
         rating = None
         if "Xmp.xmp.Rating" in m:
             rating = m["Xmp.xmp.Rating"].value
