--- a/files/usr/lib/cinnamon-settings/modules/cs_backgrounds.py	2015-06-07 17:19:26.349396798 +0000
+++ b/files/usr/lib/cinnamon-settings/modules/cs_backgrounds.py	2015-06-07 17:21:39.550391197 +0000
@@ -205,7 +205,7 @@
     def get_system_backgrounds(self):
         picture_list = []
         folder_list = []
-        properties_dir = "/usr/share/cinnamon-background-properties"
+        properties_dir = "/usr/share/gnome-background-properties"
         backgrounds = []
         if os.path.exists(properties_dir):
             for i in os.listdir(properties_dir):
