--- a/data/theme/cinnamon.css	2015-06-07 17:35:03.907357370 +0000
+++ b/data/theme/cinnamon.css	2015-06-07 17:39:50.276345327 +0000
@@ -1379,7 +1379,8 @@
 }
 .applet-box:hover .applet-icon {
 	color: #fff;
-	icon-shadow: white 0px 0px 3px;
+  /*  Broken with GNOME 3.14 (icon-size is not honoured)  
+	* icon-shadow: white 0px 0px 3px; /*
 }
 
 /* ===================================================================
--- a/files/usr/lib/cinnamon-settings/modules/cs_themes.py	2015-06-07 17:42:56.031337515 +0000
+++ b/files/usr/lib/cinnamon-settings/modules/cs_themes.py	2015-06-07 17:44:48.990332765 +0000
@@ -261,7 +261,7 @@
 
     def _load_icon_themes(self):
         dirs = ("/usr/share/icons", os.path.join(os.path.expanduser("~"), ".icons"))
-        valid = walk_directories(dirs, lambda d: os.path.isdir(d) and not os.path.exists(os.path.join(d, "cursors")) and os.path.exists(os.path.join(d, "index.theme")))
+        valid = walk_directories(dirs, lambda d: os.path.isdir(d) and os.path.exists(os.path.join(d, "index.theme")))
         valid.sort(lambda a,b: cmp(a.lower(), b.lower()))
         res = []
         for i in valid:
